src/logger.rs

```diff
@@ -1,8 +1,12 @@
 use crate::OsLog;
 use dashmap::DashMap;
+use derivative::Derivative;
 use log::{LevelFilter, Log, Metadata, Record};
 
+#[derive(Derivative)]
+#[derivative(Debug)]
 pub struct OsLogger {
+    #[derivative(Debug = "ignore")]
```

Comment:
I believe you can use [a `cfg_attr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute). ie: `#[cfg_attr(derivative, derive(Derivative))]`.

Reply:
Thanks. Derivative was used solely because that is what decorates similar structs in log4rs. I should feature gated this but did not think about that until after. I will give your suggestions a try and add a feature.